### DISCUSSION TITLE
video not available for how-to-track-football-players

### DISCUSSION THREAD

**Discussion by sainathchinta** [roboflow/notebooks#75] [Q&A]
can you upload the video for football ???

**Comment by SkalskiP** [comment]
Is there any problem with downloading from Kaggle?

**Reply by sainathchinta** [reply]
FileNotFoundError: /content/yolov5/clips/08fd33_4.mp4 does not exist

when i was running on colab it is showing like this... what to do??

**Reply by SkalskiP** [reply]
Did you download `08fd33_4.mp4` from Kaggle?

**Reply by sainathchinta** [reply]
so we need to download 08fd33_4.mp4 from kaggle 
ok i thought it will download automatically thanks a lot...
